frontend/src/operator/webui/src/app/safe-device-url.pipe.ts

```diff
@@ -10,7 +10,7 @@ export class SafeDeviceUrlPipe implements PipeTransform {
   transform(deviceId: string) {
     /* eslint-disable */
     // DO NOT FORMAT THIS LINE
-    return this.sanitizer.bypassSecurityTrustResourceUrl(`devices/${deviceId}/files/client.html`);
+    return this.sanitizer.bypassSecurityTrustResourceUrl(`/${window.location.pathname}devices/${deviceId}/files/client.html`);
```

Comment:
I expected this to be `${window.location.pathname}/devices...`.
For every page I tested, the `pathname` value starts with a `/`, but not all end with one so I expected to see `/` between `${window.location.pathname}` and `devices`. Some pages do end in `/`, but the browser normally ignores `//` and treats it like a single `/`.

Reply:
I tried to use this under proxy suburl. It seems like have a problem with window.location.pathname I'll check it further and will update the PR again.